### PR TITLE
Feature/49 make php 8 compatible

### PR DESCRIPTION
**Pull request by mediaessenz**
Solve some php 8 issues #49

### CODE REVIEW

File: Classes/Service/Extraction/AbstractExtractionService.php [hunk 1]
```diff
@@ -323,7 +325,12 @@ protected function remapServiceOutput(array $data, array $mapping = null)
 
             $value = null;
             foreach ($alternativeKeys as $dataKey) {
-                list($compoundKey, $processor) = explode('->', $dataKey);
+                if (str_contains($dataKey, '->')) {
```
Comment (xperseguers) [inline Classes/Service/Extraction/AbstractExtractionService.php:328]: This doesn't make it compatible as `str_contains` requires PHP 8, thus it breaks with PHP 7.x

Comment (mediaessenz) [reply Classes/Service/Extraction/AbstractExtractionService.php]: solved

**Comment by mediaessenz** [comment]
Add/improve some more php compatible stuff.

It would be great if you would take the changes and make them public.

The current version found at packagist and ter is not really php 8 compatible ...

**Review by xperseguers — APPROVED** [review]
thanks.

**Pull request merged**